### ISSUE TITLE
Issue with database need some help

### ISSUE THREAD

**Issue by JohnnyS** [esx-framework/esx_policejob#546]
I am getting this error in the console cannot get cars out of the garage when buying them and it doesnt show that we own them. Still fairly new to this all any help would be appreciated 

`[ERROR] [MySQL] [esx_policejob] An error happens on MySQL for query "INSERT INTO owned_vehicles (owner, vehicle, plate, type, job, `stored`) VALUES ('steam:numbersincasethisissensitive', '{\"modHydrolic\":-1,\"modFender\":-1,\"modSteeringWheel\":-1,\"modAPlate\":-1,\"modXenon\":false,\"modAerials\":-1,\"modBackWheels\":-1,\"modSideSkirt\":-1,\"modTurbo\":false,\"neonEnabled\":[false,false,false,false],\"pearlescentColor\":1,\"engineHealth\":1000.0,\"modExhaust\":-1,\"modTank\":-1,\"extras\":{\"1\":true,\"12\":true},\"fuelLevel\":30.3,\"color1\":111,\"modEngine\":-1,\"modRightFender\":-1,\"dirtLevel\":1.0,\"modDial\":-1,\"modOrnaments\":-1,\"modSpeakers\":-1,\"modTrimA\":-1,\"modSuspension\":-1,\"modWindows\":-1,\"modGrille\":-1,\"modFrame\":-1,\"modRearBumper\":-1,\"modHood\":-1,\"plate\":\"EDV 931\",\"modDoorSpeaker\":-1,\"tyreSmokeColor\":[255,255,255],\"modSeats\":-1,\"modStruts\":-1,\"modArmor\":-1,\"modFrontBumper\":-1,\"model\":1912215274,\"modDashboard\":-1,\"modSpoilers\":-1,\"neonColor\":[255,0,255],\"wheelColor\":156,\"modPlateHolder\":-1,\"windowTint\":-1,\"modShifterLeavers\":-1,\"modRoof\":-1,\"modFrontWheels\":-1,\"plateIndex\":4,\"wheels\":0,\"modArchCover\":-1,\"modVanityPlate\":-1,\"modTrunk\":-1,\"modTransmission\":-1,\"modEngineBlock\":-1,\"modHorns\":-1,\"modAirFilter\":-1,\"modBrakes\":-1,\"modLivery\":2,\"modTrimB\":-1,\"modSmokeEnabled\":false,\"color2\":0,\"bodyHealth\":1000.0}', 'EDV 931', 'car', 'police', true)": ER_NO_DEFAULT_FOR_FIELD: Field 'model' doesn't have a default value`

**Comment by Esplike** [comment]
Do you have the esx_vehicleshop installed? If so install it due to its a dependency and uses the functions of the esx_vehicleshop for the police garage shop thing.

**Comment by JohnnyS** [comment]
> Do you have the esx_vehicleshop installed? If so install it due to it's a dependency and uses the functions of the esx_vehicleshop for the police garage shop thing.

Yes I have it installed vehicle shop as well causes some issues as well. Having some issues with them per [https://github.com/esx-framework/esx_vehicleshop/issues/340](url)

**Comment by JohnnyS** [comment]
@Esplike As well this issue seems to happen with esx_ambulancejob as well. We can go buy the car but cant pull it out as it doesnt save or say we own it. Still am having this issue just put it aside

**Comment by Raydiaa** [comment]
> @Esplike As well this issue seems to happen with esx_ambulancejob as well. We can go buy the car but cant pull it out as it doesnt save or say we own it. Still am having this issue just put it aside

I have the same issue, it says the vehicle is not "mine"
